src/scripts/webview.js

```diff
@@ -68,7 +68,9 @@ class WebView extends EventEmitter {
         webviewObj.setAttribute('disablewebsecurity', 'on');
 
         webviewObj.addEventListener('did-navigate-in-page', (lastPath) => {
-            this.saveLastPath(host.url, lastPath.url);
+            if ((lastPath.url).indexOf(host.url) !== -1) {
```

Comment:
It might be easier to read using `String.prototype.includes()` here.
Was the issue just that it was saving the url and loading it next time?